src/Fixer/FunctionNotation/ImplodeCallFixer.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\FunctionNotation;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\Tokenizer\Analyzer\ArgumentsAnalyzer;
+use PhpCsFixer\Tokenizer\Analyzer\FunctionsAnalyzer;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author Kuba Werłos <werlos@gmail.com>
+ */
+final class ImplodeCallFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getDefinition()
+    {
+        return new FixerDefinition(
+            'Function `implode` must be called with 2 arguments in the documented order.',
+            [
+                new CodeSample("<?php\nimplode(\$pieces, '');\n"),
+                new CodeSample("<?php\nimplode(\$pieces);\n"),
+            ]
+        );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isCandidate(Tokens $tokens)
+    {
+        return $tokens->isTokenKindFound(T_STRING);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getPriority()
+    {
+        // must be run after NoAliasFunctionsFixer
+        return -1;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function applyFix(\SplFileInfo $file, Tokens $tokens)
+    {
+        $functionsAnalyzer = new FunctionsAnalyzer();
+
+        foreach ($tokens as $index => $token) {
+            if (!$functionsAnalyzer->isGlobalFunctionCall($tokens, $index)) {
```

Comment:
I want all the global function checking to be delegated to `FunctionsAnalyzer` class (so it can be reused e.g in `CombineNestedDirnameFixer` and here). Maybe other function `isGlobalFunctionCallWithName` or 3rd parameter to `isGlobalFunctionCall`?

Reply:
For reasoning of this PR - I see no value of extending the Analyzer.
it can be raised as idea in separated PR, which would proof it's value (eg, simplify more than one place)